Tests/AppTests/AnalyzerTests.swift

```diff
@@ -210,7 +194,7 @@ class AnalyzerTests: AppTestCase {
         XCTAssertEqual(targets.map(\.name), ["t1", "t1", "t1", "t2", "t2", "t2"])
 
         // validate score
-        XCTAssertEqual(pkg1.score, 35)
+        XCTAssertEqual(pkg1.score, 30)
```

Comment:
The score now drops, because we have `NULL` in `resolved_dependencies` where we previously had one dependency. In a real world scenario (which we're not fully testing here), this score would go back up to 35 once the build has reported back.